packages/block-library/src/cover/block.json

```diff
@@ -148,6 +148,12 @@
 		},
 		"allowedBlocks": true
 	},
+	"mediaRoles": {
+		"id": "id",
+		"url": "url",
+		"alt": "alt",
+		"type": { "attribute": "backgroundType" }
+	},
```

Comment:
If I understand this correctly, the `type` can have a hardcoded `value` or derive it from a specific attribute. Is that correct?